addon/instance-initializers/flexberry-logging.js

```diff
@@ -133,14 +103,29 @@ export function initialize(applicationInstance) {
     @private
    */
   function _mergeArguments() {
-    var ret = '';
-    for (var i = 0; i < arguments.length; i++) {
+    let ret = '';
+    for (let i = 0; i < arguments.length; i++) {
       ret += arguments[i].toString();
     }
 
     return ret;
   }
 
+  function _sendError(error) {
+    let message = error.toString();
+    let formattedMessage = JSON.stringify(
+      {
+        name: error.name,
```

Comment:
I prefer constructions like:
let errorOptions = Ember.merge({
  message: undefined,
  fileName: undefined
  ...
}, error);

And then I use "errorOptions" inside method.